CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 FEATURES:
 * r/tfe_organization: Add `allow_force_delete_workspaces` attribute to set whether admins are permitted to delete workspaces with resource under management. ([#661](https://github.com/hashicorp/terraform-provider-tfe/pull/661))
 * r/tfe_workspace: Add `force_delete` attribute to set whether workspaces will be force deleted when removed through the provider. Otherwise, they will be safe deleted. ([#675](https://github.com/hashicorp/terraform-provider-tfe/pull/675))
+* r/tfe_policy: Add OPA support for policies ([#690](https://github.com/hashicorp/terraform-provider-tfe/pull/690))
```

Comment:
This would be a good place to point out that the tfe_policy resource is new, it supports both OPA and sentinel policies, and that tfe_sentinel_policy now includes a deprecation warning